main/integration-tests/helpers/api-param-generator.js

```diff
@@ -0,0 +1,345 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+// **************** IMPORTANT  ****************
+// Please update this document if any SWB API is not found here
+// ********************************************
+
+const apiEndpoint = process.env.API_ENDPOINT;
+// const apiEndpoint = 'http://localhost:4000'; // When debugging locally
```

Comment:
Maybe this can be an input for the bash script? So tests can run against local deployment without code change.

Reply:
There's no need to treat env variables as arguments for the bash script. You can just set the env variables before running integ tests.
i.e. I can run fwoa integ tests locally with:
```
API_URL=https://xxxxxx.execute-api.us-west-2.amazonaws.com/dev COGNITO_USERNAME=workshopuser COGNITO_PASSWORD=xxx COGNITO_CLIENT_ID=xxx yarn run int-test
```

Reply:
Making it configurable in the test config file.